readme.md

```diff
@@ -376,6 +376,14 @@ Now, an act of treachery has ripped it all away, leaving her with one fragile hu
 
 _Novels concerning the end of civilisation, usually basd in a future resulting from a catastrophe of some sort, where only scattered elements of technology remain._
 
+#### [A Canticle for Leibowitz](http://www.goodreads.com/search?utf8=%E2%9C%93&query=A+Canticle+for+Leibowitz) *by Walter M. Miller Jr.* [4.0]
```

Comment:
Link to a particular book entry, instead of a search page like this one - http://www.goodreads.com/book/show/164154.A_Canticle_for_Leibowitz ?